pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
+---
+
+The AsyncAPI document is a YAML or JSON file that describes your implementation of the AsyncAPI specification. Providing a standardized way to document and describe asynchronous and event-driven systems, the AsyncAPI document generates documentation/code and defines the different components of your event-driven application, such as the events, channels, and message formats.
```

Comment:
1. Ah yes, you’re absolutely right about the first sentence being consistent in defining the AsyncAPI Document. Thank you for calling that out, I personally think that I should go more in line with: https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.16#format. 

```
The files describing the message-driven API in accordance with the AsyncAPI Specification are represented as JSON objects and conform to the JSON standards. YAML, being a superset of JSON, can be used as well to represent an AsyncAPI Specification file.
```

2. Noted, sounds like we should be going for API instead of application, per the discussion you linked.